### PR TITLE
Add shardKey header resolves #28

### LINKED ISSUE #28: Need the ability to send Ringpop Transport Header
tcurl doesn't have a --headers flag that allows us to send transport tchannel headers

### PR DESCRIPTION
**Pull request by rssathe**

### CODE REVIEW

**Comment by rf** [comment]
didn't we want to standardize on `sk` for the shard key header? @Raynos

**Comment by Raynos** [comment]
@rf yes we did...

**Comment by Raynos** [comment]
@rssathe can you make it `sk` ?

**Comment by rssathe** [comment]
@Raynos  @rf done

File: index.js [hunk 1]
```diff
@@ -204,10 +206,13 @@ function tcurl(opts) {
             return onResponse(err);
         }
 
+        var headers = opts.shardKey ? {shardKey: opts.shardKey} : {};
```
Comment (Raynos) [inline index.js]: it needs to be `{ sk: opts.shardKey }`

File: README.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ tcurl -p host:port <service> <endpoint> [options]
   Options:
     -2 [data] send an arg2 blob
     -3 [data] send an arg3 blob
+    --sk send ringpop shardKey transport header
```
Comment (Raynos) [inline README.md]: This should stay `--shardKey`

File: index.js [hunk 3]
```diff
@@ -113,6 +114,7 @@ function parseArgs(argv) {
     return {
         head: argv.head,
         body: argv.body,
+        shardKey: argv.sk,
```
Comment (Raynos) [inline index.js]: this should stay `argv.shardKey`

**Comment by rf** [comment]
lgtm

**Comment by Raynos** [comment]
Lgtm

**Pull request merged**